indra/newview/pipeline.cpp

```diff
@@ -6772,6 +6774,16 @@ void LLPipeline::generateLuminance(LLRenderTarget* src, LLRenderTarget* dst)
 			mGlow[1].bindTexture(0, channel);
 		}
 
+        channel = gLuminanceProgram.enableTexture(LLShaderMgr::DEFERRED_NORMAL);
+        if (channel > -1)
+        {
+            // bind the normal map to get the environment mask
+            mRT->deferredScreen.bindTexture(2, channel, LLTexUnit::TFO_POINT);
```

Comment:
is this normal mapping change intentional? seems unrelated to this issue

Reply:
yes, the gbuffer normal map w component contains the mask needed to tell if pixels are "sky" pixels or not